website/content/docs/concepts/domain-model/scopes.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: Domain model - scopes
-description: |-
-  Use scopes to group and manage resources in Boundary. Learn how to configure global scopes, org scopes, and projects to logically group resources.
+description: >-
+  Use scopes to group and manage resources. Learn how to configure global, org, and project scopes to logically group resources.
```

Comment:
I would consider minimizing the use of words that could be mistaken for other boundary resources, like group in this context

```suggestion
  Use scopes to organize and manage resources. Learn how to configure global, org, and project scopes to logically group resources.
```

Reply:
Good point!